cyclops-ctrl/internal/models/templates.go

```diff
@@ -39,6 +39,7 @@ type Field struct {
 	Enum          []interface{} `json:"enum"`
 	Required      []string      `json:"required"`
 	FileExtension string        `json:"fileExtension"`
+	Immutable	  bool          `json:"immutable"`
```

Comment:
Can you run 'go fmt ./...' on cyclops-ctrl?

Reply:
ran it, it downloaded a bunch of modules